src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/mha_single_token.cpp

```diff
@@ -700,6 +789,268 @@ static float dot_product(TA* a, uint8_t* b, size_t n, float* scale, float* zp, f
 #endif
 }
 
+template<typename TA, typename TB>
+static void dot_product_block(TA* a, TB* b, float* c, size_t n, size_t block_size) {
```

Comment:
Same questions. Whats the differene with previously implemented dot_product routines?

Reply:
Same reason in above.